db/schema.rb

```diff
@@ -26,4 +36,12 @@
     t.datetime "updated_at", precision: 6, null: false
   end
 
+  create_table "types", force: :cascade do |t|
```

Comment:
Why do we need a `pokemon_types` table and a `types` table? Also why isn't `type` included as a column in the main `pokemons` table?

Reply:
A Pokemon can have up to 2 types. I instinctively recoiled against the thought of having two attributes for the same thing on one model, and I caged myself in mentally by assuming we were working with SQLite.

Since we're building on PosgreSQL, I can either add the string attributes `type_1` and `type_2` to the Pokemon model instead or have a `type` attribute string array values.

What do you think?